### ISSUE TITLE
Improve the test suite

### ISSUE THREAD

**Issue by djhopper01** [dustinlakin/twitch-rb#18]
Hey @dustinlakin, would you be opposed to improving the test suite so it doesn't depend on the internet and connecting to Twitch? It looks like the test suite calls the Twitch API for each spec. Once that's done, you could hook it up to Travis CI.

I'm able to do some of this work, but I wanted to check with you before moving forward. Thanks.

**Comment by dustinlakin** [comment]
@djhopper01, I agree, and would love to see some improvement to the tests. Having CI setup on the project would be a good goal as well.
